code/game/objects/items.dm

```diff
@@ -133,7 +133,7 @@
 	src.throwing = 0
 
 	if(user.put_in_active_hand(src) && old_loc )
-		if ((user != old_loc) && (user != old_loc.get_holding_mob()))
+		if (user != old_loc.get_holding_mob())
```

Comment:
I have fixed this line of code 2x now, stop reverting the fix.

Reply:
I have no idea why this is here. What's the correct code?

Reply:
`if ((user != old_loc) && (user != old_loc.get_holding_mob()))`

Reply:
By the way, that shouldn't be here.
That should be in equipping code. 
Probably in `pre_equip` because of things like paper bin what picking up with drag and drop.

Reply:
Then it should be fixed properly in another PR.